hack/bazel-build.sh

```diff
@@ -46,9 +46,9 @@ bazel run \
 
 # linux
 bazel run \
-    --config=${ARCHITECTURE} \
```

Comment:
Note that above we already compile virtctl for the current build-platform, so if you are building on ppc64, you would still have a workabel virtctl available locally.